src/screens/AccountUnlock.js

```diff
@@ -84,8 +84,26 @@ export class AccountUnlock extends React.PureComponent {
 								pin
 							);
 						}}
-						navigate={() => {
-							if (next === 'AccountDelete') {
+						navigate={async pin => {
+							if (next === 'AccountBiometric') {
+								navigation.goBack();
+								if (accounts.getSelected().biometricEnabled) {
+									await accounts.disableBiometric(accounts.getSelectedKey());
+								} else {
+									await accounts
+										.enableBiometric(accounts.getSelectedKey(), pin)
+										.catch(error => {
+											// error here is likely no fingerprints/biometrics enrolled, so should be displayed to the user
+											console.log(error);
+											Alert.alert('Biometric Error', error.message, [
+												{
+													style: 'default',
+													text: 'Ok'
+												}
+											]);
+										});
```

Comment:
This block of code to handle `AccountBiometric` selection, better be rather wrapped into `onToggleBiometric` function like `onDelete` rather than put it here.

`AccountUnlock` Component as its name implied, it mainly contains the logic for user for unlocking the account, the action next, better be defined in where it is called, and keep this part straightforward.